app/presenters/mfa_confirmation_presenter.rb

```diff
@@ -0,0 +1,36 @@
+class MfaConfirmationPresenter
+  def initialize(user)
+    @user = user
+  end
+
+  def enforce_second_mfa?
+    IdentityConfig.store.select_multiple_mfa_options &&
+      MfaContext.new(@user).enabled_non_restricted_mfa_methods_count < 1
+  end
+
+  def heading
+    enforce_second_mfa? ?
+      I18n.t('mfa.non_restricted.heading') :
+      I18n.t('titles.mfa_setup.suggest_second_mfa')
+  end
+
+  def info
+    enforce_second_mfa? ? I18n.t(
+      'mfa.non_restricted.info_html',
+      link: learn_more,
+    ) : I18n.t('mfa.account_info', link: learn_more)
```

Comment:
I think this `mfa.acount_info` string does not have a link in it
```suggestion
    ) : I18n.t('mfa.account_info')
```